rbac/internal/views.py

```diff
@@ -582,6 +582,28 @@ def data_migration(request):
     return HttpResponse("Data migration from V1 to V2 are running in a background worker.", status=202)
 
 
+def bootstrap_pending_tenants(request):
+    """List tenants which are not bootstrapped.
+
+    GET /_private/api/utils/bootstrap_pending_tenants/
+    """
+    if request.method != "GET":
+        return HttpResponse('Invalid method only "GET" is allowed.', status=405)
+
+    public_tenant = Tenant.objects.get(tenant_name="public")
+    query = """SELECT t.org_id FROM api_tenant t
```

Comment:
Can we use django ORM: Tenant.objects.filter(tenant_mapping__isnull=True).exclude(tenant_name="public")

Reply:
I wanted to have raw sql to have this query under control, but it is same, I did little changes here and I used where clause with id, to use index. (instead of comparing on varchar field of tenant_name)

Reply:
LGTM!